src/Parsers/CommonParsers.h

```diff
@@ -542,6 +542,7 @@ namespace DB
     MR_MACROS(YY, "YY") \
     MR_MACROS(YYYY, "YYYY") \
     MR_MACROS(ZKPATH, "ZKPATH") \
+    MR_MACROS(STALENESS, "STALENESS") \
```

Comment:
Probably we should also update dictionary for fuzzer (https://github.com/ClickHouse/ClickHouse/blob/master/tests/fuzz/dictionaries/keywords.dict, I'm not sure if this is used, though)